sass/responsive/_mobile.scss

```diff
@@ -16,8 +16,18 @@
                 border: none;
             }
         }
+     .modal {
+         .settings-modal {
+             &:not(.settings-modal--tables) {
+                 &:not(.display--content) {
+                     .modal-content {
+                         background: var(--sidebar-bg);
```

Comment:
@jgilliam17, this seems to affect the background of the account settings modal in screen widths less than 768px. It actually manifests as what visually appears to be the border colour though.